src/store/messagesStore.js

```diff
@@ -875,6 +876,25 @@ const actions = {
 		}
 		return false
 	},
+
+	/**
+	 * Posts a simple text message to a room
+	 * @param {object} context default store context;
+	 * @param {strinf} token the token of the conversation to wich the message
+	 * will be forwarded;
+	 * @param {object} message the message object;
+	 */
+	async forwardMessage(context, { token, message }) {
+		const messageToBeForwarded = cloneDeep(message)
+		messageToBeForwarded.token = token
+		try {
+			const response = await postNewMessage(messageToBeForwarded)
+			return response
+		} catch (error) {
+			console.error('Error while forwarding message')
+			showError(t('spreed', 'Error while forwarding message'))
```

Comment:
Let's not catch exceptions inside the store but outside, this way we don't have UI logic in here like showError.
UI logic belongs in the Vue that calls it.